### PR TITLE
320 FAQ page

### PR DESCRIPTION
**Pull request by brmendez**
This PR includes the initial build-out of the FAQ page. Question topics are linked utilizing the `<HashLink />` component.

### CODE REVIEW

File: src/containers/Faq/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React, {FC} from 'react';
+import {HashLink} from 'components';
+
+import './Faq.scss';
+
+interface FaqContent {
+  answer: string;
```
Comment (angle943) [inline src/containers/Faq/index.tsx:7]: change this to `answer: ReactNode` (you gotta import `ReactNode` from react)

File: src/containers/Faq/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  id: string;
+  question: string;
+}
+
+const questionsAnswers: FaqContent[] = [
+  {
+    answer: 'New digital currency',
```
Comment (angle943) [inline src/containers/Faq/index.tsx:14]: now, the answer doesn't just have to be a string, but a ReactNode. So you can do something like:

`answer: <>New <A>Digital</A> currency</>,

Comment (brmendez) [reply src/containers/Faq/index.tsx:14]: Done!

File: src/containers/Faq/index.tsx (hunk 1 above)
Comment (brmendez) [reply src/containers/Faq/index.tsx]: Done!

**Review by angle943 — APPROVED** [review]

**Comment by angle943** [comment]
closes https://github.com/thenewboston-developers/Website/issues/320

**Pull request merged**